src/drivers/serac.cpp

```diff
@@ -168,10 +191,10 @@ int main(int argc, char *argv[])
   mfem::VectorCoefficient *defo_coef = new mfem::VectorFunctionCoefficient(dim, InitialDeformation);
 
   mfem::Vector velo(dim);
-  velo = 0.0;
+  velo                                       = 0.0;
```

Comment:
This seems weird. Seems like it's keying off the the line below? Not sure if we can fix this. The equal spacing was really nice above...